sdk/eventhub/azure-eventhubs/README.md

```diff
@@ -345,7 +346,7 @@ if __name__ == '__main__':
     loop = asyncio.get_event_loop()
     container_client = ContainerClient.from_connection_string(storage_connection_str, blob_name_str)
     partition_manager = BlobPartitionManager(container_client=container_client)
-    client = EventHubConsumerClient.from_connection_string(connection_str, partition_manager=partition_manager, receive_timeout=RECEIVE_TIMEOUT, retry_total=RETRY_TOTAL)
+    client = EventHubConsumerClient.from_connection_string(connection_str, event_hub_path=event_hub_path, partition_manager=partition_manager, receive_timeout=RECEIVE_TIMEOUT, retry_total=RETRY_TOTAL)
     try:
         loop.run_until_complete(client.receive(process_events, "$default"))
```

Comment:
Could we please update this sample in the same way as @lmazuel commented on the previous PR?
If I recall correctly, it was something like:
```
async def main():
    container_client = ContainerClient.from_connection_string(storage_connection_str, blob_name_str)
    partition_manager = BlobPartitionManager(container_client=container_client)
     client = EventHubConsumerClient.from_connection_string(connection_str, event_hub_path=event_hub_path, partition_manager=partition_manager, receive_timeout=RECEIVE_TIMEOUT, retry_total=RETRY_TOTAL)
    await client.receive(process_events, "$default")
    await client.close()

if __name__ == '__main__':
    loop = asyncio.get_event_loop()
    try:
        loop.run_until_complete(main())
    finally:
        loop.stop()
```

Reply:
I've updated the sample in readme